ubuntu-16.04-qt5.14.2/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:16.04
```

Comment:
What is the reason for using 16.04 instead of 18.04 or 20.04?

Reply:
See readme.